crates/cheatcodes/spec/src/vm.rs

```diff
@@ -2297,15 +2297,18 @@ interface Vm {
     function ensNamehash(string calldata name) external pure returns (bytes32);
 
     /// Returns a random uint256 value.
-    #[cheatcode(group = Utilities)]
+    /// `randomUint` is being deprecated in favor of `arbitraryUint`. It will be removed in future versions.
+    #[cheatcode(group = Utilities, status = Deprecated)]
```

Comment:
To my knowledge we don't have a system in place that emits a warning upon the use of deprecated cheatcodes, do you see a possibility to add it here?

See: https://github.com/foundry-rs/foundry/pull/8755#discussion_r1742675272

Reply:
ah, sure, thought there's already in so didn't test. Will add

Reply:
will track with https://github.com/foundry-rs/foundry/pull/8883 to keep scopes separated